packages/discovery-provider/plugins/pedalboard/apps/solana-relay/package.json

```diff
@@ -21,6 +21,8 @@
     "bs58": "4.0.1",
     "cors": "2.8.5",
     "cross-fetch": "3.1.8",
+    "dotenv": "^16.4.5",
+    "envalid": "^8.0.0",
```

Comment:
pin pls

Reply:
I don't think so, both relay plugins pull in dotenv and envalid on their own. This was the error:
```
/app/packages/discovery-provider/plugins/pedalboard/apps/solana-relay/node_modules/ts-node/src/index.ts:820
    return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/config.ts(1,20): error TS2307: Cannot find module 'dotenv' or its corresponding type declarations.
src/config.ts(2,42): error TS2307: Cannot find module 'envalid' or its corresponding type declarations.

    at createTSError (/app/packages/discovery-provider/plugins/pedalboard/apps/solana-relay/node_modules/ts-node/src/index.ts:820:12)
    at reportTSError (/app/packages/discovery-provider/plugins/pedalboard/apps/solana-relay/node_modules/ts-node/src/index.ts:824:19)
    at getOutput (/app/packages/discovery-provider/plugins/pedalboard/apps/solana-relay/node_modules/ts-node/src/index.ts:1014:36)
    at Object.compile (/app/packages/discovery-provider/plugins/pedalboard/apps/solana-relay/node_modules/ts-node/src/index.ts:1322:43)
    at Module.m._compile (/app/packages/discovery-provider/plugins/pedalboard/apps/solana-relay/node_modules/ts-node/src/index.ts:1454:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Object.require.extensions.<computed> [as .ts] (/app/packages/discovery-provider/plugins/pedalboard/apps/solana-relay/node_modules/ts-node/src/index.ts:1458:12)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19) {
  diagnosticCodes: [ 2307, 2307 ]
}
npm ERR! Lifecycle script `start` failed with error:
npm ERR! Error: command failed
npm ERR!   in workspace: @pedalboard/solana-relay@0.0.19
npm ERR!   at location: /app/packages/discovery-provider/plugins/pedalboard/apps/solana-relay
```